openedx/core/djangoapps/kafka_consumer/docs/0001_kafka_consumer_command.rst

```diff
@@ -0,0 +1,39 @@
+Managing Kafka Consumers
+--------------
+
+Status
+======
+
+In Progress
+
+Context
+=======
+As outlined in the upcoming OEP-52, we have elected to go with Apache Kafka as
```

Comment:
1. Reminder: there should be no hard wraps.
2. we => "edX.org". It may be useful to explain that although this decision is specific to edX.org, it is being shared publicly because it may be useful for others considering Kafka.